src/helics/network/zmq/ZmqCommsSS.cpp

```diff
@@ -14,9 +14,12 @@ SPDX-License-Identifier: BSD-3-Clause
 #include "../networkDefaults.hpp"
 #include "ZmqCommsCommon.h"
 #include "ZmqRequestSets.h"
-//#include <csignal>
+//#include <map>
```

Comment:
Why is this commented out?

Reply:
my guess is at one point we were using map objects and it is just left over

Reply:
It looks like this file still uses std::map, I couldn’t tell if something got messed with where new headers got inserted, since csignal is now uncommented.

Reply:
that might be how I inserted it, since the code that did that was picking off the first `#include <` and injecting it.  which probably meant that I didn't take into account comment characters.

Reply:
I fixed in the latest commit